client/Python Tests/Test_moduel_get_call.py

```diff
@@ -10,14 +25,14 @@
 class Questionnaire1Tests(unittest.TestCase):
     maxDiff = None
 
-    def test_number_of_questions(self):
+    def test_get_answer_number_of_questions(self):
```

Comment:
Follow naming convetion for tests: 
<functionUnderTest>_when_<conditionInTest>_then_<expectedResult>